.buildkite/pipelines/pull-request/packaging-tests-unix.yml

```diff
@@ -5,6 +5,33 @@ steps:
     steps:
       - label: "{{matrix.image}} / docker / packaging-tests-unix"
         key: "packaging-tests-unix-docker"
+        command: ./.ci/scripts/packaging-test.sh destructiveDistroTest.docker
+        timeout_in_minutes: 300
+        matrix:
+          setup:
+            image:
+              - debian-11
+              - opensuse-leap-15
+              - oraclelinux-7
+              - oraclelinux-8
+              - sles-12
+              - sles-15
+              - ubuntu-1804
+              - ubuntu-2004
+              - ubuntu-2204
+              - rocky-8
+              - rocky-9
+              - rhel-7
+              - rhel-8
+              - rhel-9
+              - almalinux-8
+        agents:
+          provider: gcp
+          image: family/elasticsearch-{{matrix.image}}
+          diskSizeGb: 350
+          machineType: custom-16-32768
```

Comment:
I've added the testing for default Docker image here. Done by copying the whole step to keep the steps naming. 
I was trying to do it with matrix with 2 params (similar to [this](https://github.com/elastic/elasticsearch/blob/5b1b889b97912d9c7c84b5433b26af4bfd5a797f/.buildkite/pipelines/pull-request/packaging-tests-unix-sample.yml#L1-L28)), but AFAIK that would require changing names of all steps. I am not sure how much we are attached to these names.
Let me know what you think about it.

Reply:
We're not attached to the names. I think using a 2 axis matrix makes the most sense vs duplicating the step.

Reply:
+1 on using matrix with changed names we usually don't care about.